invokeai/frontend/web/src/features/nodes/components/ui/ClearNodesButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAppDispatch } from 'app/store/storeHooks';
+import { clearNodes } from 'features/nodes/store/nodesSlice';
+import { makeToast } from 'app/components/Toaster';
+import { addToast } from 'features/system/store/systemSlice';
+import { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FaTrash } from 'react-icons/fa';
+import IAIIconButton from 'common/components/IAIIconButton';
+
+const ClearNodesButton = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const handleClearNodes = () => {
+    const confirmed = window.confirm(t('common.clearNodes'));
```

Comment:
Should use the Alert Dialog from Chakra like we have with some other components. You can reference them on how to do it.

Reply:
Ok will do

Reply:
@blessedcoolant I have updated it to use the AlertDialog from chakra-ui, I referenced it from the deleteboardsimagemodal, hope that was the right way

Reply:
![image](https://github.com/invoke-ai/InvokeAI/assets/115216705/e81f2006-f276-472e-a31c-13e9637513a0)